### ISSUE TITLE
libuv 1.27.0 breaks neovim

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#3536]
**Problem description**
Yesterdays update of libuv breaks neovim. `nvim` will exit with `1` immediately.

```
~ > gdb nvim
GNU gdb (GDB) 8.2.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "arm-linux-androideabi".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from nvim...(no debugging symbols found)...done.
(gdb) run -u NONE
Starting program: /data/data/com.termux/files/usr/bin/nvim -u NONE

Program received signal SIGSYS, Bad system call.
0xb6bfdd14 in syscall () from /system/lib/libc.so
(gdb)
```

**Steps to reproduce**
run `nvim` or `nvim -u NONE` (no configuratuin) with libuv 1.27.0

**Expected behavior**
It works when downgrading to libuv 1.26 again

**Additional information**
```
Updatable packages:
aria2/stable 1.34.0-3 arm [upgradable from: 1.34.0-2]
mpd/stable 0.21.5 arm [upgradable from: 0.21.4]
newsboat/stable 2.14.1-1 arm [upgradable from: 2.14.1]
System information:
Linux localhost 3.4.0-perf-g284400aea4b #1 SMP PREEMPT Mon Feb 25 13:31:32 UTC 2019 armv7l Android
Termux-packages arch:
arm
Android version:
8.1.0
Device manufacturer:
Fairphone
Device model:
FP2
```
```
03-18 10:41:41.554  5944  5944 F libc    : Fatal signal 31 (SIGSYS), code 1 in tid 5944 (nvim), pid 5944 (nvim)
03-18 10:41:41.609  5947  5947 I crash_dump32: obtaining output fd from tombstoned, type: kDebuggerdTombstone
03-18 10:41:41.614  5947  5947 I crash_dump32: performing dump of process 5944 (target tid = 5944)
03-18 10:41:41.614  5944  5944 F libc    : failed to resend signal during crash: Operation not permitted
03-18 10:41:41.575  5947  5947 W crash_dump32: type=1400 audit(0.0:77596): avc: denied { search } for name="com.termux" dev="mmcblk0p20" ino=246018 scontext=u:r:crash_dump:s0:c512,c768 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
03-18 10:41:41.595  5947  5947 I chatty  : uid=10084(com.termux.api) crash_dump32 identical 7 lines
03-18 10:41:41.595  5947  5947 W crash_dump32: type=1400 audit(0.0:77604): avc: denied { search } for name="com.termux" dev="mmcblk0p20" ino=246018 scontext=u:r:crash_dump:s0:c512,c768 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
03-18 10:41:41.595  5947  5947 W crash_dump32: type=1400 audit(0.0:77605): avc: denied { search } for name="home" dev="mmcblk0p20" ino=253906 scontext=u:r:crash_dump:s0:c512,c768 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
03-18 10:41:41.595  5947  5947 W crash_dump32: type=1400 audit(0.0:77606): avc: denied { search } for name="com.termux" dev="mmcblk0p20" ino=246018 scontext=u:r:crash_dump:s0:c512,c768 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
03-18 10:41:41.595  5947  5947 W crash_dump32: type=1400 audit(0.0:77607): avc: denied { search } for name="home" dev="mmcblk0p20" ino=253906 scontext=u:r:crash_dump:s0:c512,c768 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
03-18 10:41:41.625  5947  5947 E DEBUG   : unexpected waitpid response: n=5944, status=00000100
03-18 10:41:41.615  5861  5861 W zsh     : type=1400 audit(0.0:77608): avc: denied { signull } for scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:r:crash_dump:s0:c512,c768 tclass=process permissive=0
```

**Comment by yubrajbhoi** [comment]
@Neo-Oli How did you downgrade the package ? Please respond.

**Comment by Neo-Oli** [comment]
@yubrajbhoi I rebuilt it by recompiling with this commit: 6a87477985e570f958e57bc7feaaa1f3869ecf7f. It's the one just before the upgrade.

**Comment by fornwall** [comment]
I got `disallowed system call 291`, which should be the [statx](http://man7.org/linux/man-pages/man2/statx.2.html) system call, which indeed [seems to be used by libuv 1.27.0](https://github.com/libuv/libuv/commit/a4fc9a66cc35256dbc4dcd67c910174f05b6daa6).

**Comment by fornwall** [comment]
This is now fixed in the updated `1.27.0-1` version of libuv, which is available for installation.
